controllers/scopeinstance_controller.go

```diff
@@ -455,3 +462,30 @@ func hashScopeInstanceAndTemplate(si *operatorsv1.ScopeInstance, st *operatorsv1
 
 	return util.HashObject(hashObj)
 }
+
+func updateStatusScopeTemplateNotFound(in *operatorsv1.ScopeInstance, err error) {
+	meta.SetStatusCondition(&in.Status.Conditions, metav1.Condition{
+		Type:    operatorsv1.TypeScoped,
+		Status:  metav1.ConditionFalse,
+		Reason:  operatorsv1.ReasonScopeTemplateNotFound,
+		Message: fmt.Sprintf("getting ScopeTemplate %q: %s", in.Spec.ScopeTemplateName, err),
+	})
+}
+
+func updateStatusScopingFailed(in *operatorsv1.ScopeInstance, err error) {
+	meta.SetStatusCondition(&in.Status.Conditions, metav1.Condition{
+		Type:    operatorsv1.TypeScoped,
+		Status:  metav1.ConditionFalse,
+		Reason:  operatorsv1.ReasonScopingFailed,
+		Message: err.Error(),
+	})
+}
+
+func updateStatusScopingSuccessful(in *operatorsv1.ScopeInstance, msg string) {
+	meta.SetStatusCondition(&in.Status.Conditions, metav1.Condition{
+		Type:    operatorsv1.TypeScoped,
+		Status:  metav1.ConditionTrue,
+		Reason:  operatorsv1.ReasonScopingSuccessful,
+		Message: msg,
+	})
+}
```

Comment:
Not sure these were really needed. They seem simple enough you could've just left inline. Do not change it for this PR, but I'm okay with these being inline in future.